yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -43,11 +43,6 @@ class FlowableOperator : public Flowable<D> {
         : flowable_(std::move(flowable)), subscriber_(std::move(subscriber)) {
       assert(flowable_);
       assert(subscriber_);
-
-      // We expect to be heap-allocated; until this subscription finishes (is
-      // canceled; completes; error's out), hold a reference so we are not
-      // deallocated (by the subscriber).
-      Refcounted::incRef(*this);
```

Comment:
With these refcounted types, if another object has a handle to it, it should never be destroyed; passing around a Reference rather than a raw pointer fixes that. Having to manually inc/dec the refcount is a bug.